### ISSUE TITLE
The conformance tool webpage needs to handle inline comments

### ISSUE THREAD

**Issue by nslowell** [anweiss/cddl#205]
As the RFC says:

> Comments are started by a ";" (semicolon) character and finish at the end of a line (LF or CRLF)

but the conformance tool at https://cddl.anweiss.tech/ throws errors when they exist.

```
Version = [
    major: u32, ; comment
    minor: u32,
    patch: u32,
]
```

Results in:

> invalid group entry syntax
> 
> missing definition for rule Version

**Comment by anweiss** [comment]
@nslowell this has been fixed in #206. The website has also been updated.

**Comment by nslowell** [comment]
Hurray!
